examples/colors/main.rs

```diff
@@ -0,0 +1,194 @@
+use std::cell::RefCell;
+use std::rc::Rc;
+
+use gtk::glib::clone;
+use gtk::prelude::*;
+
+mod color;
+mod store;
+
+use color::Color;
+use store::ColorStore;
+
+fn main() {
+    let application = gtk::Application::new(
+        Some("com.github.gtk-rs.examples.apps_launcher"),
+        Default::default(),
+    );
+
+    application.connect_activate(build_ui);
+    application.run();
+}
+
+fn build_ui(app: &gtk::Application) {
+    let window = gtk::ApplicationWindow::builder()
+        .default_width(600)
+        .default_height(400)
+        .application(app)
+        .title("Colors")
+        .build();
+
+    let header = gtk::HeaderBar::new();
+    window.set_titlebar(Some(&header));
+
+    let model = ColorStore::new(8);
+
+    let sort_model = gtk::SortListModel::new(Some(&model), None::<&gtk::Sorter>);
```

Comment:
```suggestion
    let sort_model = gtk::SortListModel::new(Some(&model), gtk::Sorter::NONE);
```
same thing elsewhere